src/components/common/PostalCode/PostalCodeBar.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Button from 'src/components/ui/Button'
+import Icon from 'src/components/ui/Icon'
+import './postal-code.scss'
```

Comment:
You're already importing it in the proper file so there's no need to import it here as well.

https://github.com/vtex-sites/base.store/blob/aa15d19fc7d9a30cadb4618878b58951951b8034/src/styles/global/components.scss#L9

```suggestion
```